docs-src/scim/index.rst

```diff
@@ -0,0 +1,97 @@
+==============================================
+SCIM API Client
+==============================================
+
+`SCIM API <https://api.slack.com/scim>`_ is a set of APIs for provisioning and managing user accounts and groups. SCIM is used by Single Sign-On (SSO) services and identity providers to manage people across a variety of tools, including Slack.
+
+`SCIM (System for Cross-domain Identity Management) <http://www.simplecloud.info/>`_ is supported by myriad services. It behaves slightly differently than other Slack APIs.
```

Comment:
```suggestion
`SCIM (System for Cross-domain Identity Management) <http://www.simplecloud.info/>`_ is supported by a myriad of services. It behaves slightly differently from other Slack APIs.
```

Reply:
This part is taken from https://api.slack.com/scim - if we change this part, we can change the original one too

Reply:
It also sounded wrong to me, but supposedly it is acceptable to use myriad without of. https://www.dictionary.com/e/how-do-you-use-the-word-myriad/

Reply:
Thanks, y'all! I've applied Alissa's suggestion to this page. Let's discuss whether we can change the original sentence separately.